qemu/tests/cfg/ceph_image_mem_leak.cfg

```diff
@@ -0,0 +1,67 @@
+- ceph_image_mem_leak:
+    only ceph
+    virt_test_type = qemu
+    type = ceph_image_mem_leak
+    start_vm = yes
+    kill_vm = yes
+    images += " disk1 disk2 disk3 disk4 disk5 disk6 disk7 disk8 disk9 disk10"
+    # disk1
+    image_name_disk1 = images/disk1
+    image_size_disk1 = 1G
+    remove_image_disk1 = yes
+    force_create_image_disk1 = yes
```

Comment:
remove_image = yes
remove_image_image1 = no
force_create_image = yes
force_create_image_image1 = no

Then we don't need to set these for all data images
But the current is OK for me

Reply:
Updated, thanks.